R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -10,6 +10,23 @@ vignette: >
   \usepackage[utf8]{inputenc}
 ---
 
+<!--
```

Comment:
I have worked with a few Apache projects that do this in doc though.
I understand we don't. I'm fine either way but since the JIRA is opened there really isn't any file without license except for DESCRIPTION (doesn't support comment, already has a license tag) and NAMESPACE (which is really a build file, and very often tool generated).